src/components/ZoomableImage/zoomedImage.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { StyledBackdrop, StyledCenteredImage, StyledImageContainer } from './styled';
+import getTransform from './helpers/getTransform';
+
+const ZoomedImage = ({ src, alt, close, originalRect }) => {
+    const [isCentered, setIsCentered] = useState(false);
+
+    useEffect(() => {
+        setIsCentered(true);
```

Comment:
why not init state to true instead?

Reply:
because we need to render the image in its original position and then animate to zoom, and isCentered controls this